notebooks/06 Modelling - classification.ipynb

```diff
@@ -726,7 +729,7 @@
     "    if model == \"catboost\":\n",
```

Comment:
When you pipeline this work as a whole I feel like a lot of the catboost duplication will be able to be collapsed down. We can revisit when we come to pipeline.